app/src/main/java/com/project/gamedb/ultis/ImageExtensions.kt

```diff
@@ -4,13 +4,25 @@ import android.widget.ImageView
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.CenterCrop
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.bumptech.glide.request.RequestOptions
 import com.project.gamedb.R
 
 fun ImageView.loadImage(urlImage: String) {
     Glide.with(context)
         .load(urlImage)
+        .apply(RequestOptions().override(this.width, this.height))
         .placeholder(R.drawable.image_loading)
         .error(R.drawable.image_error)
         .transform(CenterCrop(), RoundedCorners(25))
         .into(this)
 }
+
+fun ImageView.setPlatform(platform: String) {
+    when (platform) {
+        Constants.PC -> this.setImageResource(R.drawable.ic_pc)
+        Constants.NINTENDO -> this.setImageResource(R.drawable.ic_nintendo)
+        Constants.PLAYSTATION -> this.setImageResource(R.drawable.ic_playstation)
+        Constants.XBOX -> this.setImageResource(R.drawable.ic_xbox)
+    }
+    if (platform == Constants.ANDROID || platform == Constants.IOS) this.setImageResource(R.drawable.ic_phone)
```

Comment:
Không cần phải thêm this
```suggestion
        Constants.XBOX -> setImageResource(R.drawable.ic_xbox)
        Constants.ANDROID, 
        Constants.IOS -> setImageResource(R.drawable.ic_phone)
    }
```